[nrf noup] boot: Use NCS_ prefix for sdk-nrf specific Kconfigs

de-nordic opened the pull request:
nrf-squash! [nrf noup] boot: Add retry for image verification

Use NCS_ prefix, for Kconfigs added in noups, to avoid collisions. The NRF_ prefix will remain reserved for Kconfigs related to products or sdk-nrf subsystems.

de-nordic committed:
[nrf noup] boot: Use NCS_ prefix for sdk-nrf specific Kconfigs

nrf-squash! [nrf noup] boot: Add retry for image verification

Use NCS_ prefix, for Kconfigs added in noups, to avoid collisions.
The NRF_ prefix will remain reserved for Kconfigs related
to products or sdk-nrf subsystems.

Signed-off-by: Dominik Ermel <dominik.ermel@nordicsemi.no>

ahasztag commented:
Do you also want to change: https://github.com/nrfconnect/sdk-mcuboot/blob/4e48cbdb3bce5e24b12fda26731dafcef4079d26/boot/zephyr/Kconfig#L425 ?

de-nordic commented:
> Do you also want to change:
> 
> https://github.com/nrfconnect/sdk-mcuboot/blob/4e48cbdb3bce5e24b12fda26731dafcef4079d26/boot/zephyr/Kconfig#L425
> ?

Yes. I am quite sure it was not there when I did the PR.

de-nordic committed:
[nrf noup] boot: Use NCS_ prefix for sdk-nrf specific Kconfigs

nrf-squash! [nrf noup] Added BOOT_SIGNATURE_USING_ITS for ecdsa configuration

Replace NRF_BOOT_SIGNATURE_USING_ITS with NCS_BOOT_SIGNATURE_USING_ITS prefix.

Signed-off-by: Dominik Ermel <dominik.ermel@nordicsemi.no>
